### ISSUE TITLE
inconsistent-args misunderstands anonymous objects

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#737]
This is my code:

```
[n] > app
  malloc.of > @
    8
    [m]
      while > @
        m.get.eq 42 >> [i]
        m.put 42 >> [i]
```

I'm getting:

```
app:8 Object 'app.ξ.ρ.m.put' has arguments inconsistency (the usage clashes with []) (inconsistent-args (WPA))
app:7 Object 'app.ξ.ρ.m.get.eq' has arguments inconsistency (the usage clashes with []) (inconsistent-args (WPA))
```

Looks like a false positive warning to me.

**Comment by bibonix** [comment]
I am planning to work on this issue. The lint `inconsistent-args` (WPA) appears to flag uses inside `>>` (anonymous-object decoration) of method calls with arguments, treating the abstract `[i]` decorator as the application form, which then "clashes" with the real call `(m.get.eq 42)` / `(m.put 42)`. I will trace through `LtInconsistentArgs` and the supporting XSL to confirm and propose a fix with a regression YAML pack.

**Comment by bibonix** [comment]
@yegor256 update on my earlier note: I traced through the code and the `inconsistent-args` lint (along with all the other WPA lints) was extracted from this repository in commit 235cda2d (#845, `Refactor XMIR analysis classes for simplicity.`) on 2026-04-24, and now lives in [`objectionary/wpa`](https://github.com/objectionary/wpa). It is no longer reachable from `Source` in this repo, so issue #737 — which describes a false positive produced by `LtInconsistentArgs` — is no longer reproducible against `objectionary/lints`. The issue (if still reproducible against `wpa`) should probably be re-filed there. I am dropping this and switching to a different bug.
